src/PLTBinaryFileReader.cc

```diff
@@ -429,13 +430,13 @@ void PLTBinaryFileReader::ReadOnlinePixelMask(const std::string maskFileName, co
 
     for (int col = firstCol; col <= lastCol; ++col) {
       for (int row = firstRow; row <= lastRow; ++row) {
-	if (maskVal == 0) {
-	  // fPixelMask contains the masked pixels. So if maskVal == 0 (means "turn this pixel off").
-	  // put this pixel into fPixelMask, and if it's 1, take this pixel out of fPixelMask.
-	  fPixelMask.insert(ch*100000 + roc*10000 + col*100 + row);
-	} else {
-	  fPixelMask.erase(ch*100000 + roc*10000 + col*100 + row);
-	}	  
+
```

Comment:
Not sure why you deleted this comment here -- can you put it back? Also the printf probably isn't necessary any more.